pages/common/conda.md

```diff
@@ -0,0 +1,35 @@
+# conda
+
+> Package, dependency and environment management for any programming language.
+
+- Create a new environment, installing named packages into it:
+
+`conda create --name {{environment_name}} {{python=2.7}} {{matplotlib}}`
```

Comment:
I think it would be more semantically meaningful to write `{{python=2.7 matplotlib}}`, since they're part of the same token (packages). Do you agree?